### ISSUE TITLE
Unable to import Akamai property - ExactlyOne error

### ISSUE THREAD

**Issue by clstokes** [pulumi/pulumi-akamai#30]
#### Problem 

As reported at https://stackoverflow.com/questions/65852453/input-validation-fails-when-importing-an-akamai-property-into-pulumi, trying to `pulumi import pulumi import akamai:index/property:Property ...` fails with the error below.

```
Diagnostics:
  pulumi:pulumi:Stack (akamai-import-dev):
    error: preview failed

  akamai:index:Property (example_property):
    error: akamai:index/property:Property resource 'example_property' has a problem: ExactlyOne: "group_id": one of `group,group_id` must be specified
    error: akamai:index/property:Property resource 'example_property' has a problem: ExactlyOne: "contract_id": one of `contract,contract_id` must be specified
    error: Preview failed: one or more inputs failed to validate
```

#### Workaround

Using the import resource option as described at https://www.pulumi.com/docs/guides/adopting/import/#pulumi-import-resource-operation should allow the resource to be imported and avoid this error.

**Comment by Milananas** [comment]
We are experiencing the same issue, is there already an update regarding this problem?

**Comment by stack72** [comment]
We believe this has been fixed and will be available in v2.7.0 of the pulumi-akamai provider
